.env.example

```diff
@@ -3,3 +3,6 @@ TOKEN=
 
 # The ID of the guild the bot will run in
 GUILD=
+
+# Channel to send welcome/leave messages in
+GATEWAY_CHANNEL=
```

Comment:
As you had already mentioned in Discord, we might want to implement this in the db for more flexibility. Now that the db is ready (we just need reviewers), if we do decide to use the db rather than an env value, you should wait for the db pr to be merged and then integrate that.

Reply:
I would personally not add these variables in the ENV file and keep the ENV file for sensitive data like the TOKEN or the bot ID, and make an implementation later to be able to change these in runtime, maybe in the config database schema created in the other PR.